### ISSUE TITLE
Matching of empty line in JSDoc comments

### ISSUE THREAD

**Issue by rubycon** [robmisasi/eslint-plugin-headers#1]
Here a small reproductible case. The plugin is setup to add the content of this file at the beginning of each file :

```txt
Copyright (c) 1998, Regents of the University of California
All rights reserved.

Redistribution and use in source and binary forms, with or without...
```

However, subsequent use of eslint won't match the comment inserted by the plugin :
```js
/**
 * Copyright (c) 1998, Regents of the University of California
 * All rights reserved.
 *
 * Redistribution and use in source and binary forms, with or without...
 */
```

The issue seems to come from the comment matcher which always expect a trailing space on empty lines for JSDoc comments.

The matcher shouldn't assume the presence of a trailing space because most editors have a trim on save option. Moreover, the plugin comment formatter also trim trailing space when inserting comments.

**Comment by robmisasi** [comment]
Thank you for providing this example and explanation, I will work to reproduce this behavior and provide an update within the next few days.

**Comment by robmisasi** [comment]
@rubycon I've addressed this issue in v1.0.4, please let me know if you experience further issues.

**Comment by rubycon** [comment]
Thank you very much. It fixes the issue for me!
